### ISSUE TITLE
Error with OpsWorks and Amazon Linux 2015.03

### ISSUE THREAD

**Issue by tomalessi** [fog/fog#3522]
I just upgraded to Amazon Linux 2015.03 and one of my chef jobs that configures Route53 is failing with the following error:

*NameError: uninitialized constant Fog::AWS::CredentialFetcher*

I believe this relates to fog using v1 of the aws-ask gem instead of v2, as detailed here:

http://ruby.awsblog.com/post/TxFKSK2QJE6RPZ/Upcoming-Stable-Release-of-AWS-SDK-for-Ruby-Version-2

Is anyone else experiencing this issue?

**Comment by geemus** [comment]
We don't use the aws-sdk gem, so I don't think it should be that. Could you report this against fog/fog-aws and add details of what versions you have and a stack trace? Thanks!

**Comment by geemus** [comment]
P.S. I think this might actually be a duplicate of fog/fog-aws#83

**Comment by bshehram** [comment]
I had this exact same issue. Using OpsWorks with the latest 03.2015 Amazon Linux AMI. I have my custom cookbook that is calling route53 0.4.0. I tried the fixes listed above what finally got me working was forcing the version of fog to be 1.33.0 (the current latest). I did this by putting in the following line in my custom cookbook's attributes/default.rb file:

default['route53']['fog_version'] = '1.33.0'

Cheers!
